### PR TITLE
Add formatting check to .travis.yml

### PR DESCRIPTION
**Pull request by polymetric1**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,15 @@ language: rust
 rust:
   - nightly
 sudo: false
+before_script:
```
Comment (skylerberg) [inline .travis.yml]: You need to create the formatted folder first. `mkdir formatted`.

**Comment by skylerberg** [comment]
See inline comment.

In general this is not a big issue, but if you want to make your pull requests prettier you can do an interactive rebase and squash commits that do not add to the history. For example " attempt to fix cp command again" could be squashed.

**Comment by polymetric1** [comment]
Yeah, I will definitely do that. I keep recommitting to test the travis thing, but I realize i shouldn't have to do that. I'm still trying to figure out a way to get the `cp` command to work

**Comment by skylerberg** [comment]
the problem is that `formatted` does not exist.

You need to do
```
mkdir formatted
cp -r * formatted/

**Comment by polymetric1** [comment]
I know, but that still causes an error and halts the build.

**Comment by polymetric1** [comment]
I'll still try it, though, I just haven't had a chance to

**Comment by polymetric1** [comment]
I'm going to close this PR and create a new one when I have this figured out.

**Pull request closed without merging**